composer.json

```diff
@@ -21,6 +21,9 @@
         "yiisoft/yii2": "*",
         "twig/twig": "~1.0"
     },
+    "require-dev": {
+        "phpunit/phpunit": "3.7.*"
+    },
```

Comment:
I'd like to keep this off from the extensions because dev setup of yii would have phpunit installed a dozen times in vendor which is not needed,